docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,46 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v2/users?offset=0&limit=10&guests=false
+
+including an ``access_token`` of a server admin.
+The parameters ``offset`` and ``limit`` are required only for pagination.
+Per default a ``limit`` of 100 is used. If the endpoint returns less entries
+than specified by ``limit`` then there are no more users left.
+The parameter ``name`` can be used to filter by user name.
```

Comment:
it seems to be filtering by user_id rather than by (display) name. The `users` table has a `name` column, but that's a historical error; we refer to user_id elsewhere.